locales/en/index.yml

```
@@ -2061,6 +2061,8 @@ faq:
     title: "My card has no security code."
     content: "Check well on the back (for Maestro, Mastercard, Visa) or on the front (for American Express cards). If you don't find any security code, most likely your card is not enabled for online payments. Contact your bank for more information."
 bonus:
+  accessibility:
+    sectionCardLabel: "Bonus and discounts"
```

```suggestion
    sectionCardLabel: "Update bonus and discounts"
```
For the same reason of the previous comment.